test/panel/group-filtering.spec.tsx

```diff
@@ -15,7 +15,7 @@ import server from '../../src/tasks/preset/server';
 it('should allow filtering of tasks (client only)', () => {
   // 1: Initial render
   const channel: Channel = new Channel({ async: false });
-  const { container, queryByText } = render(
+  const { container, queryByText, debug } = render(
```

Comment:
I don't think `debug` is needed now

Reply:
Sorry about this, my editor probably forgot to warn me about it